task6/task6.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+border_char=""
+string_to_print=""
+
+while getopts ":s:c:" opt; do
+  case $opt in
+    s) string_to_print="$OPTARG" ;;
+    c) border_char="$OPTARG" ;;
+    \?) echo "Invalid option: -$OPTARG" >&2
+        exit 1 ;;
+  esac
+done
+
+string_length=${#string_to_print}
+
+if [ "$border_char" != "" ]; then
+  
+  border=""
+  for ((i=0; i<string_length+4; i++)); do
+      border="${border}${border_char}"
+  done
+
+
+  echo "$border"
+
+  echo "${border_char} $string_to_print ${border_char}"
```

Comment:
don't put "{}" braces around your variables in the string. Just the "$" dollar sign will do the work.